src/js/tabs/sensors.js

```diff
@@ -429,9 +452,29 @@ sensors.initialize = function (callback) {
 
                     samples_mag_i = addSampleToData(mag_data, samples_mag_i, FC.SENSOR_DATA.magnetometer);
                     drawGraph(magHelpers, mag_data, samples_mag_i);
-                    raw_data_text_ements.x[2].text(FC.SENSOR_DATA.magnetometer[0].toFixed(2));
-                    raw_data_text_ements.y[2].text(FC.SENSOR_DATA.magnetometer[1].toFixed(2));
-                    raw_data_text_ements.z[2].text(FC.SENSOR_DATA.magnetometer[2].toFixed(2));
+
+                    const magX = FC.SENSOR_DATA.magnetometer[0];
+                    const magY = FC.SENSOR_DATA.magnetometer[1];
+                    const magZ = FC.SENSOR_DATA.magnetometer[2];
+                    const magXSquared = Math.pow(FC.SENSOR_DATA.magnetometer[0], 2);
+                    const magYSquared = Math.pow(FC.SENSOR_DATA.magnetometer[1], 2);
+                    const magZSquared = Math.pow(FC.SENSOR_DATA.magnetometer[2], 2);
+                    const magSumSqR = Math.sqrt(magXSquared + magYSquared + magZSquared).toFixed(0);
+                    const magCalibrationX = magCalX(magX);
+                    const magCalibrationY = magCalY(magY);
+                    const magCalibrationZ = magCalZ(magZ);
+
+                    samples_mag_i = addSampleToData(mag_data, samples_mag_i, FC.SENSOR_DATA.magnetometer);
+                    drawGraph(magHelpers, mag_data, samples_mag_i);
+
+//                    raw_data_text_ements.x[2].text(FC.SENSOR_DATA.magnetometer[0].toFixed(2));
```

Comment:
Do we still need these comments here?

Reply:
no, not really, they can go away.  Just kept them to remember the original code.  
I'll remove them when the consensus is to merge or not merge.
What we really need is a dedicated 'Mag' tab, similar in concept to the dedicated 'GPS' tab, where we can perform calibrations, check raw values, etc.